### PR TITLE
fix(detect): Detect _TZ3000_empogkya and _TZ3000_lubfc1t5 as TuYa ZM-L03E-Z and M10Z

### PR DESCRIPTION
**Pull request by felipejfc**

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -2180,6 +2178,8 @@ const definitions: Definition[] = [
         },
         whiteLabel: [
             tuya.whitelabel('Lonsonho', 'X703A', '3 Gang switch with backlight', ['_TZ3000_rhkfbfcv']),
+            tuya.whitelabel('Zemismart', 'ZM-L03E-Z', '3 gang switch with neutral', ['_TZ3000_empogkya']),
+            tuya.whitelabel('TuYa', 'M10Z', '2 gang switch with 20A power socket', ['_TZ3000_lubfc1t5']),
```
Comment (Koenkk) [inline src/devices/tuya.ts:2182]: > 2 gang switch

This definition is for a 3 gang switch

Comment (felipejfc) [reply src/devices/tuya.ts:2182]: This device is a 2 Gang with a power socket that is also threated as a switch (same controls). Regardless, I will split them

Comment (felipejfc) [reply src/devices/tuya.ts:2182]: Do you think we need to split?

**Comment by felipejfc** [comment]
Added image: https://github.com/Koenkk/zigbee2mqtt.io/pull/2267

File: src/devices/tuya.ts (hunk 1 above)
Comment (Koenkk) [reply src/devices/tuya.ts:2182]: Then this is fine!

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**